### PR TITLE
[Konvoy] Publish 1.6.0-beta docs

### PR DESCRIPTION
**Pull request by samvantran**
## Description of changes being made
This publishes the first 1.6.0-beta.0 docs for Konvoy

## Checklist
- [x] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [x] Create your PR against `staging`, not `master`. 
- [x] Update all links if you are moving a page.
- [x] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/ksphere/konvoy/1.6.0-beta/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+layout: layout.pug
+navigationTitle: Konvoy 1.5
+title: Welcome to Konvoy 1.5
```
Comment (samvantran) [inline pages/ksphere/konvoy/1.6.0-beta/index.md:4]:
```suggestion
navigationTitle: Konvoy 1.6-beta
title: Welcome to Konvoy 1.6-beta
```

File: pages/ksphere/konvoy/1.6.0-beta/addons/dependencies/index.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+navigationTitle: Dependencies
+title: Addon Dependencies
+menuWeight: 7
+excerpt: Understand addon dependencies and interactions
+enterprise: false
+---
+
+<!-- markdownlint-disable MD018 -->
+
+When deploying Konvoy, addons are a integral part of the cluster's functionality. When deploying or troubleshooting addons, it helps to understand how addons interact and may require other addons as dependencies.
```
Comment (russurquhart1) [inline pages/ksphere/konvoy/1.6.0-beta/addons/dependencies/index.md:12]:
```suggestion
When deploying Konvoy, addons play an integral part of the clusters functionality. When deploying or troubleshooting addons, it helps to understand how addons interact and may require other addons as dependencies.
```

File: pages/ksphere/konvoy/1.6.0-beta/addons/dependencies/index.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+If an addon's dependency does not successfully deploy, the addon requiring that dependency does not successfully deploy. Addon dependencies are determined by their `.spec.requires` key value pair item.
```
Comment (russurquhart1) [inline pages/ksphere/konvoy/1.6.0-beta/addons/dependencies/index.md:14]:
```suggestion
If an addon's dependency does not deploy, the addon requiring that dependency does not deploy. The addon dependencies `.spec.requires` key value pair item decides their dependencies.
```

File: pages/ksphere/konvoy/1.6.0-beta/addons/dependencies/index.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+For example, the cert-manager addon is a dependency of traefik. If we take a look at the traefik `ClusterAddon` object, we see the following `.spec.requires` key value pair item:
```
Comment (russurquhart1) [inline pages/ksphere/konvoy/1.6.0-beta/addons/dependencies/index.md:16]:
```suggestion
For example, traefix has the cert-manager addon as a dependency. The traefik `ClusterAddon` object has the following `.spec.requires` key value pair item:
```

File: pages/ksphere/konvoy/1.6.0-beta/addons/dependencies/index.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+```yaml
+   - matchLabels:
+        kubeaddons.mesosphere.io/name: cert-manager
+```
+
+As indicated by the key `matchLabels`, the addon looks for its dependencies using labels on objects.
+
+If we then view the `.metadata.labels` key value pair item of the cert-manager `ClusterAddon` object, we see the following label:
```
Comment (russurquhart1) [inline pages/ksphere/konvoy/1.6.0-beta/addons/dependencies/index.md:25]:
```suggestion
The `.metadata.labels` key value pair item of the cert-manager `ClusterAddon` object, has the following label:
```

File: pages/ksphere/konvoy/1.6.0-beta/addons/dependencies/index.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+```yaml
+   kubeaddons.mesosphere.io/name: cert-manager
+```
+
+In this example, if the cert-manager addon does not deploy, the traefik addon does not deploy.
+
+In addition to understanding why an addon does not deploy, these concepts help troubleshoot an addon not working properly. An addon dependency could not be healthy and cause issues. The table below lists Konvoy addons and their dependencies.
```
Comment (russurquhart1) [inline pages/ksphere/konvoy/1.6.0-beta/addons/dependencies/index.md:33]:
```suggestion
These concepts help troubleshoot an addon not deploying or working. Addon dependencies can cause issues. The table below lists Konvoy addons and their dependencies.
```

File: pages/ksphere/konvoy/1.6.0-beta/addons/dependencies/index.md (hunk 2 above)
Comment (cmastr) [reply pages/ksphere/konvoy/1.6.0-beta/addons/dependencies/index.md:12]: @russurquhart1 this isn't grammatically correct.

**Comment by cmastr** [comment]
closing in favor of #3191

**Pull request closed without merging**